src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -788,6 +791,9 @@ internal async Task InvokeResponse(InvocationResponse invokeResponse)
             if (_executingInvocations.TryRemove(invokeResponse.InvocationId, out ScriptInvocationContext context)
                 && invokeResponse.Result.IsInvocationSuccess(context.ResultSource, capabilityEnabled))
             {
+                _metricsLogger.LogEvent(string.Format(MetricEventNames.WorkerInvokeSucceeded, Id));
+                _workerChannelLogger.LogInformation("Received successful invocation response for invocationId: {invocationId} and workerId: {workerId}", invokeResponse.InvocationId, Id);
```

Comment:
same question / nit here:
```suggestion
                _workerChannelLogger.LogTrace("Received successful invocation response for invocationId: {invocationId} and workerId: {workerId}", invokeResponse.InvocationId, Id);
```